packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -89,10 +162,33 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
 function remoteDocumentsStore(
   txn: PersistenceTransaction
 ): SimpleDbStore<DbRemoteDocumentKey, DbRemoteDocument> {
+  return getStore<DbRemoteDocumentKey, DbRemoteDocument>(
+    txn,
+    DbRemoteDocument.store
+  );
+}
+
+/**
+ * Helper to get a typed SimpleDbStore for the documentChangesStore object store.
+ */
+function documentChangesStore(
+  txn: PersistenceTransaction
+): SimpleDbStore<DbDocumentChangesKey, DbDocumentChanges> {
+  return getStore<DbDocumentChangesKey, DbDocumentChanges>(
+    txn,
+    DbDocumentChanges.store
+  );
+}
+
+/**
+ * Helper to get a typed SimpleDbStore from a transaction.
+ */
+function getStore<KeyType extends IDBValidKey, ValueType>(
```

Comment:
Might be worth making this a static function on SimpleDbTransaction?

Reply:
Done